src/shallow.js

```diff
@@ -11,13 +12,31 @@ function nodeToJson(node) {
         return node;
     }
 
+    if (!isElement(node)) {
+        if (Array.isArray(node)) {
+            return node.map(nodeToJson);
+        }
+
+        if (typeof node === 'object') {
+            const obj = {};
+
+            Object.keys(node).forEach(key => {
+                obj[key] = nodeToJson(node[key]);
+            });
+
+            return obj;
```

Comment:
I would prefer that to use `Object.entries` and `reduce`:
```js
return Object.entries(node).reduce((obj, [key, val]) => {
    obj[key] = nodeToJson(val);
    return obj;
}, {});
```

What do you think?